src/styles/layout.css

```diff
@@ -494,3 +494,31 @@ a.question::before {
 h2:has(inlinecode) {
   line-height: 1.5;
 }
+
+/* Kapa.ai overrides */
+#kapa-widget-container .mantine-Button-root {
+  z-index: 1000;
+}
+
+.mantine-Image-root {
+  height: 2rem;
+}
+
+.mantine-Text-root {
```

Comment:
@nilubava, I also added these overrides, I thought it'd be easier than asking you to tweak things... :)

However, the result is not exactly what I had locally through making these exact changes in the inspector. I attached a screenshot. Could you help with fixing, based on the lines I added and this picture?

I thought the smaller size fits better, and I made the border radius consistent with our other similar elements.

![CleanShot 2023-09-01 at 19 59 55](https://github.com/prisma/docs/assets/7353770/dd40b5bb-7831-4942-bb71-fd830635f5d6)

Reply:
@petradonka I have added some `!important` tags to force these styles.

Reply:
Easy solution 😄 Thanks!